shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -83,10 +83,10 @@
    * Interface for those objects that maintain and expose a reference to a {@code FlutterView} (such
    * as a full-screen Flutter activity).
    *
-   * <p>This indirection is provided to support applications that use an activity other than {@link
-   * io.flutter.app.FlutterActivity} (e.g. Android v4 support library's {@code FragmentActivity}).
-   * It allows Flutter plugins to deal in this interface and not require that the activity be a
-   * subclass of {@code FlutterActivity}.
+   * <p>This indirection is provided to support applications that use an activity other than
```

Comment:
does this class also need to be deleted?

Reply:
Yes. The intention is to remove this in multiple PRs in order of safety. Removing FlutterView also requires heavily modifying/removing plugin registry classes as they are dependent on this.

Reply:
oh ok